src/LSTM/index.js

```diff
@@ -19,19 +19,23 @@ const regexWeights = /weights|weight|kernel|kernels|w/gi;
 const regexFullyConnected = /softmax/gi;
 
 class LSTM {
-  constructor(model, callback) {
+  constructor(modelPath, callback) {
     this.ready = false;
     this.model = {};
     this.cellsAmount = 0;
+    this.cells = [];
+    this.zeroState = { c: [], h: [] };
+    this.state = { c: [], h: [] };
     this.vocab = {};
     this.vocabSize = 0;
     this.defaults = {
-      seed: 'a',
+      seed: 'a', // TODO: use no seed by default
       length: 20,
       temperature: 0.5,
+      stateful: false,
```

Comment:
ok sure, though this will not be totally backwards compatible in terms of results, since if you reuse the same (ml5js) LSTM class without zeroing the state, you might get a different result. 
e.g. 
when stateless, if we do:
1. lstm.generate({seed:"the meaning of life is "}); // will zero state, seed, and then run inference
2. lstm.generate({seed:"the meaning of life is "}); // again will zero state, seed, and then run inference, and produce identical result to #1

whereas with stateful, the second lstm.generate(..) will continue from the previous state, i.e. will be the equivalent of lstm.generate({seed:"the meaning of life is <whatever was sampled> the meaning of life is "}.

I don't know which is better for the default, but I just wanted to raise this little warning :)

(P.S. this reminds me, that a `resetState()` methpd might be useful to reset the state to zero)